apps/ejabberd/src/ejabberd_s2s_out.erl

```diff
@@ -899,13 +899,13 @@ handle_info({send_element, El}, StateName, StateData) ->
             case catch Mod:Fun(El) of
```

Comment:
I don't remember any use of it. Maybe @kzemek knows more about it since his working with s2s more recently.